src/extension.ts

```diff
@@ -306,6 +317,40 @@ export function activate(context: VSCode.ExtensionContext) {
   );
 
   context.subscriptions.push(openRuleCommand);
+
+  const updateServerStorageCommandCallback = () => {
+    const params: ExecuteCommandParams = {
+      command: updateServerStorageCommandName,
+      arguments: getSonarLintConfiguration().get(connectedModeServersSectionName)
+    };
+    languageClient.sendRequest(ExecuteCommandRequest.type, params).then(
+      success => {
+        VSCode.window.showInformationMessage("Successfully updated SonarLint server storage");
+      },
+      reason => {
+        VSCode.window.showWarningMessage("Failed to update SonarLint server storage");
+      }
+    );
+  };
+  const updateServerStorageCommand = VSCode.commands.registerCommand(updateServerStorageCommandName, updateServerStorageCommandCallback);
+  context.subscriptions.push(updateServerStorageCommand);
+
+  const updateProjectBindingCommandCallback = () => {
+    const params: ExecuteCommandParams = {
+      command: updateProjectBindingCommandName,
+      arguments: getSonarLintConfiguration().get(connectedModeProjectSectionName)
+    };
+    languageClient.sendRequest(ExecuteCommandRequest.type, params).then(
+      success => {
+        VSCode.window.showInformationMessage("Successfully updated SonarLint project binding");
+      },
+      reason => {
+        VSCode.window.showWarningMessage("Failed to update SonarLint project binding");
+      }
+    );
+  };
+  const updateProjectBindingCommand = VSCode.commands.registerCommand(updateProjectBindingCommandName, updateProjectBindingCommandCallback);
+  context.subscriptions.push(updateProjectBindingCommand);
```

Comment:
Is it not problem that you can still execute the project binding command even if the previous command failed ?

Reply:
@gregaubert Errors may be raised to the user, depending on what he changed and what happens. Christ might have feedback on usability, I'll see with him when he gets there, I don't have a particular concern at this point..